start/resources/requirements.md

```diff
@@ -74,44 +71,36 @@ Below is a list of common attributes that you can include in your submit file `r
 - **OSGVO_OS_VERSION** - Version of the operating system
 
 - **OSGVO_OS_STRING** - Combined OS name and version. Common values are
-  _RHEL 6_ and _RHEL 7_. Please see the requirements string above on the
+  _RHEL 7_ and _RHEL 8_. Please see the requirements string above on the
   recommended setup.
 
 - **OSGVO_CPU_MODEL** - The CPU model identifier string as presented in
   /proc/cpuinfo
-- **CVMFS_oasis_opensciencegrid_org_REVISION** - Attribute specifying
-  the need to access specific oasis /cvmfs file system repositories. See the
-  example above for determining the appropriate REVISION number when using
-  this `requirements` attribute.
+
+- **HAS_CVMFS_oasis_opensciencegrid_org** - Attribute specifying
+  the need to access specific oasis /cvmfs file system repositories.
 
 - **CUDACapability** - For GPU jobs, specifies the CUDA compute capability.
```

Comment:
```suggestion
- **CUDACapability** - For GPU jobs, specifies the CUDA compute capability.
See our [GPU guide](5000653025) for more details. 
```

Reply:
Added this as a commit as github would not let me direct commit it.